CONTRIBUTING.md

```diff
@@ -72,6 +73,8 @@ For instructions on reporting security issues and bugs, please see [security][se
 
 For questions about building, running, or troubleshooting, start with the [Developer's Guide][dev-guide], and work your way through the process that we've outlined. If that doesn't answer your question(s), try to post on [Discussion][discussion-link] tab or if you think you found a bug, please file an [issue][issue-link].
```

Comment:
Fix paragraph indent (seems to be right shifted at least in the review)

Reply:
It's just in the review.

Reply:
Looks good here https://github.com/mahalrs/orkestra/blob/contributing-add-slack/CONTRIBUTING.md#support